### PR TITLE
Add TypeScript definition

### PR DESCRIPTION
**Pull request by zhongyuanjia**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for creating a TypeScript definition. 🙌 Can you follow [this styleguide](https://github.com/sindresorhus/typescript-definition-style-guide)?

Some examples:
- https://github.com/sindresorhus/semver-regex/commit/039944bb5b3876913171b895488874b5c200e458
- https://github.com/sindresorhus/global-dirs/commit/0aae4d64887f77af94c4fb7ca12aa9f93c0a8291
- https://github.com/sindresorhus/md5-hex/commit/bbbfeff840f26973fbffc16d4863ae6d120079f3

**Comment by zhongyuanjia** [comment]
@sindresorhus OK, it has been modified

**Comment by sindresorhus** [comment]
You need to fix the relevant lint violations.

File: package.json [hunk 1]
```diff
@@ -35,6 +38,7 @@
 	],
 	"devDependencies": {
 		"ava": "*",
+		"tsd": "^0.14.0",
```
Comment (sindresorhus) [inline package.json:39]: You're not actually running this.

**Comment by zhongyuanjia** [comment]
So strange，xo always show this
image

**Pull request merged**